src/test/java/seedu/address/testutil/AttendanceUtil.java

```diff
@@ -0,0 +1,15 @@
+package seedu.address.testutil;
+
+import seedu.address.logic.commands.AddAttendanceCommand;
+import seedu.address.model.student.Attendance;
+
+public class AttendanceUtil {
+    /**
+     * Returns an add command string for adding the {@code Attendance}.
+     */
+    public static String getAddAttendanceCommand(Attendance attendance) {
+        String correctArgument = " 1 week/2";
+        return AddAttendanceCommand.COMMAND_WORD + " " + correctArgument;
```

Comment:
I think you meant to exclude one of the spaces, if not you will get a double space after the COMMAND_WORD.